### ISSUE TITLE
(GoogleChrome) Unable to install latest version

### ISSUE THREAD

**Issue by gep13** [chocolatey/chocolatey-coreteampackages#962]
Looks like the checksums of the latest GoogleChrome package no longer match what is being downloaded.

Can we push out a new version of this package?

![image](https://user-images.githubusercontent.com/1271146/36588195-3a1a3360-187f-11e8-8a22-0d0fd48b3c37.png)

It works fine when using the Download Cache, so something obviously changed on the Google side.

**Comment by gep13** [comment]
Noticed this issue after seeing this issue on the AppVeyor site:

https://github.com/appveyor/ci/issues/2089

**Comment by gep13** [comment]
@AdmiringWorm is all that is required here likely to be forcing a new AppVeyor run to check for latest version?

**Comment by filipesilva** [comment]
Yeah seeing the same thing on AppVeyor at the moment. Just doing a new AppVeyor does not seem to be enough.

**Comment by gep13** [comment]
Hmm, looks like the problem is bigger than I had hoped 😢 

Looks like the site that we use to get the latest version of Chrome isn't working:

![image](https://user-images.githubusercontent.com/1271146/36588504-3c5ee8fe-1880-11e8-954e-30133ed99008.png)

**Comment by gep13** [comment]
Do we know who runs that site, and who we can contact to figure out what is going on?

**Comment by AdmiringWorm** [comment]
seems like the website is up again, and a new version have been pushed to chocolatey.
When it have been approved I believe we can close this issue.

**Comment by gep13** [comment]
@AdmiringWorm am I right in saying that there wasn't any output from AU to say that the website wasn't working properly, when it was down?  Would be good if we could catch this happening again.

**Comment by AdmiringWorm** [comment]
@gep13 not completely sure, but yeah I don't think there was any output from AU, could be that the ignore messages catches it instead.

**Comment by AdmiringWorm** [comment]
hmm,  I stand corrected.
AU actually detected the error, must have missed the email instead:
https://ci.appveyor.com/project/chocolatey/chocolatey-coreteampackages/build/3422#L292

**Comment by gep13** [comment]
Ah, I think I see why I didn't see it...

Looks like it failed to update the gist:

https://ci.appveyor.com/project/chocolatey/chocolatey-coreteampackages/build/3422#L458

Hence it wasn't in the report.

Hopefully your new fix will help with that as well.

**Comment by AdmiringWorm** [comment]
true, but it probably got sent out as an email instead.

Yeah, hopefully that fix will help with the reports, so far so good though.

**Comment by gep13** [comment]
> @AdmiringWorm said...
true, but it probably got sent out as an email instead.

Yip.  I think I must have missed that in the flood of other emails that I have 😄

**Comment by AdmiringWorm** [comment]
same here
